composer.json

```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
-        "illuminate/config": ">=5.5 <6.0",
-        "illuminate/support": ">=5.5 <6.0",
-        "illuminate/database": ">=5.5 <6.0"
+        "php": ">=7.1.3",
```

Comment:
This is not valid, because `>=` includes next major `8.x` version and it could not work.

Reply:
Indeed, I've corrected.